vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet.cc

```diff
@@ -239,15 +240,23 @@ void Wallet::DisconnectAllWallets(ledger::ResultCallback callback) {
   callback(type::Result::LEDGER_OK);
 }
 
-type::BraveWalletPtr Wallet::GetWallet() {
-  const std::string wallet_string =
-      ledger_->ledger_client()->GetEncryptedStringState(state::kWalletBrave);
+type::BraveWalletPtr Wallet::GetWallet(bool create) {
+  auto json = ledger_->state()->GetEncryptedString(state::kWalletBrave);
+  if (!json)
+    return nullptr;
+
+  if ((*json).empty()) {
```

Comment:
Nit: Any reason not to just use `json->empty()` here?

Reply:
Nope - thanks!